misk-aws/src/main/kotlin/misk/jobqueue/sqs/MetricsExt.kt

```diff
@@ -0,0 +1,13 @@
+package misk.jobqueue.sqs
+
+import misk.metrics.Histogram
+import misk.time.timed
+
+/**
+ * Time the work
+ */
+fun <T> Histogram.timeCall(vararg labelValues: String, work: () -> T): T {
```

Comment:
Should I added to the Misk histogram instead of using this extension?

Reply:
The only concern of moving this to `Histogram` is we're explicitly recording milliseconds in this method. Whereas, Histogram doesn't know what time unit we are using. What do you think?

Reply:
Good point! Maybe `timedMills`?

I don't imagine we'll want to use this functionality with anything other than millis (i.e. very low chance of ending up with many `timed*` methods in this interface) but if we want to make a conscious decision to keep `Histogram` generic, we can also accept a `java.time.temporal.ChronoUnit`, e.g.:
```
histogram.timed(ChronoUnit.MILLIS, "foo", "bar") { work() }
```

Reply:
Done